e2e-tests/tfm/cypress/fixtures/mock-gef-facilities.js

```diff
@@ -18,9 +36,7 @@ exports.MOCK_FACILITY_ONE = {
     'Other',
   ],
   detailsOther: 'Other',
-  currency: {
-    id: 'GBP',
-  },
+  currency: 'GBP',
```

Comment:
Looking at Staging facility data this was correct previously. I think `currency: 'GBP` is  a BSS/EWCS thing

Reply:
reverted